src/core/router/testsroute.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Route, RouteProps, useHistory } from 'react-router-dom';
+import { testsContext } from 'core/tests-context';
+
+export const TestsRoute: React.FunctionComponent<RouteProps> = props => {
+  const { testInProgress } = React.useContext(testsContext);
+  const history = useHistory();
+
+  React.useEffect(() => {
+    console.log(testInProgress);
```

Comment:
We should remove this console log.

On the other hand what happens if we move directly to a settings page? I think is fine to move to home page in that case but don't know if in the future this "works by side effect" could lead us to some issues